chore: editorial review (#111)
KILTprotocol/docs: docs/develop/01_sdk/01_quickstart.md

import Disconnect from '!!raw-loader!@site/code_examples/core_features/getting_started/09_disconnect.ts';

The following guide will give you a starting point to begin with KILT.
What you will have achieved at the end of this is the following:

1. Import the **KILT SDK** into a project
2. Connect to the **KILT blockchain**
3. Query a **web3name** to get its **DID**
4. Verify a **Credential**, published via a **DID service endpoint**

> After completing the quickstart guide, you should have learn a little about KILT by actually using it.

Reviewer: ```suggestion
After completing the quickstart guide, you should have learned a little about KILT by actually using it.
```

Now you are able to access the SDK and all its functionality.
We will move onto connecting to the **KILT blockchain**.

### Connect to the KILT Blockchain

Connecting and disconnecting to the KILT blockchain is required for any operation that relies on the KILT blockchain, such as **querying and verifying a Credential**.

Still within the same `main` function, you need to configure the SDK to connect to a **KILT node**.
For this, the SDK exposes **`Kilt.init()`** to configure the address of the node to connect to.

We will use the official **Spiritnet** address:

<SnippetBlock
  className="language-js"
>
  {InitSDK}
</SnippetBlock>

Once the node address has been configured, we can establish a connection with the Spiritnet node.
Again, the SDK makes this process very easy to follow, by exposing a `Kilt.connect()` function:

<SnippetBlock
  className="language-js"
>
  {Connect}
</SnippetBlock>

> Now you have connected you have access to the chain, but let's not forget to **close** any connections when we are done!

Reviewer: ```suggestion
After connecting you have access to the chain, but let's not forget to **close** any connections when we are done!
```

  {FetchDid}
</SnippetBlock>

Try executing it and check the result.

Did you get the DID? Nice work! You now have `john_doe`'s DID.
Now let's see if John Doe has any public KILT Credentials that we could retrieve and verify!

## Retrieve and Verify a Credential

A **KILT DID** can expose service endpoints, which allow to link external resources to the DID.
One type of external resources is represented by, you guessed it, **KILT Credentials**!
Therefore, let's see how we can retrieve the **service endpoints** of John Doe's DID and see if it links to any public Credentials for us to **query** and **verify**.

We will keep adding code below what we just added.

> The code snippet retrieved the service endpoints exposed by the DID we found for `john_doe`:

Reviewer: ```suggestion
The code snippet retrieves the service endpoints exposed by the DID we found for `john_doe`:
```
